ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -349,7 +371,7 @@ static int component_select(struct ompi_win_t *win, void **base, size_t size, in
                                    &(mca_osc_ucx_component.ucp_worker));
         if (UCS_OK != status) {
             OSC_UCX_VERBOSE(1, "ucp_worker_create failed: %d", status);
-            ret = OMPI_ERROR;
+            ret = OMPI_ERR_TEMP_OUT_OF_RESOURCE;
```

Comment:
looks like `_osc_ucx_init_unlock()` will not be called in this case (and a few error cases below)

Reply:
@brminich  thanks, and done, I forced pushed the changes; however. I'm not sure if that's allowed or if it required a new commit

Reply:
@janjust that is the right way